agents/droidlet_agent.py

```diff
@@ -3,29 +3,74 @@
 """
 import logging
 import random
+import re
+import sentry_sdk
+import spacy
 import time
 import numpy as np
 import datetime
 import os
+from typing import Dict
 
 from agents.core import BaseAgent
 from agents.scheduler import EmptyScheduler
-from droidlet.shared_data_structs import ErrorWithResponse
-from droidlet.interpreter import InterpreterBase
+from droidlet.base_util import hash_user
 from droidlet.event import sio, dispatch
+from droidlet.interpreter import InterpreterBase, coref_resolve, process_spans_and_remove_fixed_value
 from droidlet.memory.save_and_fetch_commands import *
+from droidlet.shared_data_structs import ErrorWithResponse
 
 random.seed(0)
+spacy_model = spacy.load("en_core_web_sm")
 
 DATABASE_FILE_FOR_DASHBOARD = "dashboard_data.db"
 DEFAULT_BEHAVIOUR_TIMEOUT = 20
 MEMORY_DUMP_KEYFRAME_TIME = 0.5
+
```

Comment:
can we put this either in a util or in dialogue_manager.py (and import here)?

Reply:
Not sure why this should go under `dialogue_manager` ?

Reply:
some nsp util is fine too (am pointing to the postprocess_logical_form function)